firestore-bigquery-export/extension.yaml

```diff
@@ -184,21 +187,43 @@ params:
         value: eu      
     default: us
     required: true
-    immutable: true  
+    immutable: true 
+
+  - param: BIGQUERY_PROJECT_ID
+    label: Project Id
+    description: >-
+      Override the default project bigquery instance. 
+      This can allow updates to be directed to a bigquery instance on another project.
```

Comment:
```suggestion
    label: BigQuery Project ID
    description: >-
      Optionally override the default project ID. 
      This can allow updates to be directed to a BigQuery instance on another project.
```